tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/tests/PitclipsePitMutationsViewTest.java

```diff
@@ -33,16 +34,22 @@ public static void setupJavaProject() throws CoreException {
 
     @Test
     public void selectMutationOpensTheClassAtTheRightLineNumber() throws CoreException {
-        runPackageTest(FOO_BAR_PACKAGE, TEST_PROJECT);
-        coverageReportGenerated(2, 80, 0, 6, 0);
-        PitclipseSteps pitclipseSteps = new PitclipseSteps();
-        PitMutation mutation = fromMutationLine(
-        "SURVIVED    | " + TEST_PROJECT + " | foo.bar | foo.bar.Foo |    7 | negated conditional");
-        pitclipseSteps.doubleClickMutationInMutationsView(mutation);
-        pitclipseSteps.mutationIsOpened(FOO_CLASS + ".java", 7);
-        mutation = fromMutationLine(
-        "SURVIVED    | " + TEST_PROJECT + " | foo.bar | foo.bar.Bar |    7 | negated conditional");
-        pitclipseSteps.doubleClickMutationInMutationsView(mutation);
-        pitclipseSteps.mutationIsOpened(BAR_CLASS + ".java", 7);
+        final long timeOutBefore = SWTBotPreferences.TIMEOUT;
+        try {
+            SWTBotPreferences.TIMEOUT = 10000;
```

Comment:
Because this timeout value is used for how long we are waiting for the pit run. Currently, the timeout is set to 5 Seconds and I turned it up to 10 seconds to give pit enough time to finish.

Should I use the search timeout instead, but this seems to be very long?

Reply:
I think that we should use the same timeout consistently and make it configurable from the outside as we do right now... @echebbi what do you think?

Reply:
> I think that we should use the same timeout consistently and make it configurable from the outside as we do right now

@LorenzoBettini I'm not sure I understood you, what do you mean exactly by "make it configurable from the outside"? As you said above the timeout is already configurable through the `org.eclipse.swtbot.search.timeout` property.

I agree that we should keep the timeout as consistent as possible thoughout the tests and avoiding changing it programmatically. Ideally I'd actually like to have a dedicated property that defines the timeout before we stop to wait for PIT so that we can change it without impacting the timeout for UI elements; something like `org.pitest.pitclipse.tests.pit.timeout`, but we'll see that it in another PR.

@JKutscha it looks like you reverted this change in your latest commit and the build seems to succeed so I guess this conversation can be closed? If you do need to increase the timeout, we set it [here](https://github.com/pitest/pitclipse/blob/157167faca6044b311f4a0606d0184770019ba45/tests/org.pitest.pitclipse.ui.tests/pom.xml#L89) for the Maven build:

https://github.com/pitest/pitclipse/blob/157167faca6044b311f4a0606d0184770019ba45/tests/org.pitest.pitclipse.ui.tests/pom.xml#L89

Reply:
> > I think that we should use the same timeout consistently and make it configurable from the outside as we do right now
> 
> @LorenzoBettini I'm not sure I understood you, what do you mean exactly by "make it configurable from the outside"? As you said above the timeout is already configurable through the `org.eclipse.swtbot.search.timeout` property.

Sorry for the misunderstanding: I meant "keep it" as it is right now :)

> 
> I agree that we should keep the timeout as consistent as possible thoughout the tests and avoiding changing it programmatically. Ideally I'd actually like to have a dedicated property that defines the timeout before we stop to wait for PIT so that we can change it without impacting the timeout for UI elements; something like `org.pitest.pitclipse.tests.pit.timeout`, but we'll see that it in another PR.

Yes, that should be trivial to implement. If you have time please open an issue or I'll do that ASAP

Reply:
> If you have time please open an issue or I'll do that ASAP

Done, see #178